code/controllers/subsystem/vote.dm

```diff
@@ -407,3 +421,21 @@ SUBSYSTEM_DEF(vote)
 
 	if(SSvote)
 		SSvote.browse_to(client)
+
+/**
+  * Determine if the station has power.
+  *
+  * Actually, we'll just cheat and see if the Particle Accelerator is active.
+  * If the Particle Accelerator is active, we'll assume that we have power.
+  * See: https://www.youtube.com/watch?v=-dJolYw8tnk
+  */
+/proc/i_have_the_power()
```

Comment:
For the Honor of Greyskull!

Maybe we should change this to --- nah, who am I kidding xD

Reply:
https://www.youtube.com/watch?v=rOd-iMiqh9A